java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java

```diff
@@ -196,4 +203,438 @@ CompletableFuture<Double> geodist(
      * }</pre>
      */
     CompletableFuture<String[]> geohash(String key, String[] members);
+
+    /**
+     * Returns the members of a sorted set populated with geospatial information using {@link
+     * #geoadd(String, Map)}, which are within the borders of the area specified by a given shape.
+     *
+     * @since Redis 6.2.0 and above.
```

Comment:
Remove reference to Redis and refer to Valkey instead.

Reply:
This is a good point but all the older commands also mention redis, I think there will be a large PR that does all the replacing after most commands get done, so I guess I will do this if there are other changes need addressing, or else I think it can be skipped for now.